Marlin/language_fr_utf8.h

```diff
@@ -200,28 +200,28 @@
   #define MSG_VB_JERK                       _UxGT("Vb jerk")
   #define MSG_VC_JERK                       _UxGT("Vc jerk")
 #else
-  #define MSG_VA_JERK                       _UxGT("Vx jerk")
-  #define MSG_VB_JERK                       _UxGT("Vy jerk")
-  #define MSG_VC_JERK                       _UxGT("Vz jerk")
+  #define MSG_VX_JERK                       _UxGT("Vx jerk")
+  #define MSG_VY_JERK                       _UxGT("Vy jerk")
+  #define MSG_VZ_JERK                       _UxGT("Vz jerk")
 #endif
 #define MSG_VE_JERK                         _UxGT("Ve jerk")
 #define MSG_VELOCITY                        _UxGT("Vélocité")
-#define MSG_VMAX                            _UxGT("Vmax")
-#define MSG_VMIN                            _UxGT("Vmin")
+#define MSG_VMAX                            _UxGT("Vmax ")
+#define MSG_VMIN                            _UxGT("Vmin ")
 #define MSG_VTRAV_MIN                       _UxGT("V dépl. min")
 #define MSG_ACCELERATION                    _UxGT("Accélération")
 #define MSG_AMAX                            _UxGT("Amax ")
 #define MSG_A_RETRACT                       _UxGT("A retrait")
-#define MSG_A_TRAVEL                        _UxGT("A Dépl.")
+#define MSG_A_TRAVEL                        _UxGT("A dépl.")
 #define MSG_STEPS_PER_MM                    _UxGT("Pas/mm")
 #if IS_KINEMATIC
   #define MSG_ASTEPS                        _UxGT("A pas/mm")
   #define MSG_BSTEPS                        _UxGT("B pas/mm")
   #define MSG_CSTEPS                        _UxGT("C pas/mm")
 #else
-  #define MSG_ASTEPS                        _UxGT("X pas/mm")
-  #define MSG_BSTEPS                        _UxGT("Y pas/mm")
-  #define MSG_CSTEPS                        _UxGT("Z pas/mm")
+  #define MSG_XSTEPS                        _UxGT("X pas/mm")
+  #define MSG_YSTEPS                        _UxGT("Y pas/mm")
+  #define MSG_ZSTEPS                        _UxGT("Z pas/mm")
```

Comment:
all is ok , it's just  some syntax errors
You can merge , you can trust 
It worked before with 'en ' , it will work with ' fr '
Before  ' lcd shown  a b  c ' , now show ' x y z ' 
It's all , nothing dramatic and no big impact on the firmware
No risk of armageddon like THE PAUSE PR THAT HAVE ALL DESTROY
Please look my post in the issue tab ' There is someone that you trust and you merge all PR of this guy SINCE ONE YEAR ' , and every week i see the pause always modified , never tested , only made for kinematic , and always some tons of fix to made after this guy 
Please , take time and see the shit !!!
Go on my issue POST

Reply:
Ok, so I'll do it for you then.